pycaption/scc/__init__.py

```diff
@@ -212,8 +212,12 @@ def read(self, content, lang='en-US', simulate_roll_up=False, offset=0):
         :type offset: int
         :param offset:
 
+        :type src_fps: float
```

Comment:
Please update the user documentation as well with a short explanation and a usage example for both reader and writer class. You can find the documentation in `docs/supported_formats.rst` at line 120